File: c#/src/Docmost.Sdk/Models/CreateAdminUserDto.cs
namespace Docmost.Sdk.Models;

public class CreateAdminUserDto
{
    [System.Text.Json.Serialization.JsonPropertyName("name")]
    public string Name { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("email")]
    public string Email { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("password")]
    public string Password { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("workspaceName")]
    public string? WorkspaceName { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("hostname")]
    public string? Hostname { get; set; }

}

File: go/docmost/models/create_admin_user_dto.go
package models

// CreateAdminUserDto from OpenAPI.
type CreateAdminUserDto struct {
	Name string `json:"name"`
	Email string `json:"email"`
	Password string `json:"password"`
	WorkspaceName *string `json:"workspaceName,omitempty"`
	Hostname *string `json:"hostname,omitempty"`
}
